shotover-proxy/tests/cassandra_int_tests/basic_driver_tests.rs

```diff
@@ -0,0 +1,65 @@
+use crate::cassandra_int_tests::CassandraTestContext;
+use crate::helpers::ShotoverManager;
+use test_helpers::docker_compose::DockerCompose;
+
+use cassandra_cpp::*;
+use serial_test::serial;
+use std::thread;
+use tracing::debug;
+
+fn test_create_keyspace(ctx: CassandraTestContext) {
+    let mut query = stmt!(
+        "CREATE KEYSPACE IF NOT EXISTS cycling WITH REPLICATION = { 'class' : 'SimpleStrategy', 'replication_factor' : 1 };"
+    );
+
+    let mut result = ctx.session.execute(&query).wait().unwrap();
+    debug!("{:?} query result {:?}", thread::current().id(), result);
+    assert_eq!(result.row_count(), 0);
+
+    query = stmt!("SELECT release_version FROM system.local");
+    result = ctx.session.execute(&query).wait().unwrap();
+    debug!("{:?} query result {:?}", thread::current().id(), result);
+    assert_eq!(result.row_count(), 1);
+    assert_eq!(
+        result
+            .first_row()
+            .unwrap()
+            .get_column(0)
+            .unwrap()
+            .get_str()
+            .ok(),
+        Some("3.11.10")
```

Comment:
Currently the error is thrown away and not displayed when a failure occurs.
Instead we should write like this so we get a panic that displays the error.
```rust
    assert_eq!(
        result
            .first_row()
            .unwrap()
            .get_column(0)
            .unwrap()
            .get_str()
            .unwrap(),
        "cycling"
    );
 ```